src/scripts/main.js

```diff
@@ -1 +1,60 @@
 'use strict';
+
+const body = document.querySelector('body');
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
```

Comment:
Add event listener on document, not on logo.

Reply:
Done